### PR TITLE
Okazaki - Follow Up - P1

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

**Review by clsackett — APPROVED** [review]

File: src/gdcdictionary/schemas/follow_up.yaml [hunk 1]
```diff
@@ -1908,17 +1909,23 @@ properties:
     $ref:
       - "_terms.yaml#/imaging_findings/common"
     enum:
+      - Bladder Involvement
       - Carcinomatosis
       - Distant Metastasis
       - Equivocal
+      - Extra-Pelvic Metastatic Disease
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/follow_up.yaml:1916]: what should the capitalization be like per our rules?
"Extra-pelvic Metastatic Disease" (Jira)
or "Extra-Pelvic Metastatic Disease" (the PR)

edit: guessing the PR is the correct one

Comment (wwysoc2) [reply src/gdcdictionary/schemas/follow_up.yaml:1916]: Per the style guide:  "For hyphenated words, e.g. Iliac-External, all elements should be capitalized unless they are articles (a, an, the), prepositions (in, on, off, out, etc.) or coordinating conjunctions (and, but, for, or, nor)."

So yes, the PR is the correct one.  JIRA is now fixed.

**Review by ebusygina — APPROVED** [review]

**Review by wteouchicago — APPROVED** [review]

File: src/gdcdictionary/schemas/follow_up.yaml [hunk 2]
```diff
@@ -1894,6 +1894,7 @@ properties:
     items:
       enum:
         - Abdomen
+        - Cervix Uteri
```
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/follow_up.yaml:1897]: the capitalization does not match the ticket.

**Review by stilesj-uchicago — COMMENTED** [review]
I had one question regarding capitalization.

File: src/gdcdictionary/schemas/follow_up.yaml (hunk 2 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/follow_up.yaml:1897]: Fixed in JIRA ticket, should be title case

**Review by stilesj-uchicago — APPROVED** [review]

**Pull request merged**